tests/FSProcedureLoaderTest.php

```
@@ -0,0 +1,35 @@
+<?php
+
+/* Copyright (c) 2016 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see ./LICENSE */
+
+require_once(__DIR__."/../classes/autoload.php");
+
+use CaT\Plugins\Jill\RPC;
+
+class FSProcedureLoaderTest extends PHPUnit_Framework_TestCase {
+	public function setUp() {
+		$this->loader = new RPC\FSProcedureLoader(__DIR__."/rpc");
+	}
+
+	public function test_proc_exists() {
+		$exist = $this->loader->procedureExists("testProc");
+		$this->assertTrue($exist);
+
+		$exist = $this->loader->procedureExists("testProcParams");
+		$this->assertTrue($exist);
+	}
+
+	public function test_proc_not_exists() {
+		$exist = $this->loader->procedureExists("testProcNotThere");
+		$this->assertFalse($exist);
+	}
+
+	public function test_load_proc() {
+		$proc = $this->loader->loadProcedure("testProc");
+		$this->assertInstanceOf(RPC\Procedure::class, $proc);
+
+		$proc = $this->loader->loadProcedure("testProcParams");
+		$this->assertInstanceOf(RPC\Procedure::class, $proc);
+	}
+	
```

Maybe write test on `$this->loader->loadProcedure("testProcNotThere")`. This is also undefined behaviour in the interface. I would expect it to throw an `\InvalidArgumentException` in that case.